test/spectral/tests/info-and-hint.test.js

```diff
@@ -1,60 +0,0 @@
-const path = require('path');
```

Comment:
Was this file deleted intentionally?

Reply:
I renamed that file to `spectral-config.test.js` since I wanted to expand it to cover the new configuration options, but I guess I crossed some threshold of similarity since git now thinks I deleted and added a file.  But you can see the the original "info-and-hint" test is there in the new file.